flake8-check.sh

```diff
@@ -0,0 +1,4 @@
+set -e
+FLAKE8_ARGS="--count --max-line-length=120 --statistics"
+flake8 ${XMNR_DIR}/python ${XMNR_DIR}/test/unit/ ${FLAKE8_ARGS} --show-source
```

Comment:
maybe extra FLAKE8_CMD variable that has all the common parts of following two lines?